setup.cfg

```diff
@@ -45,5 +45,5 @@ max-line-length = 160
 docstring-convention=google
 
 [tool:pytest]
-addopts = --cov mbed_targets --cov-report xml:coverage/coverage.xml
+addopts = --junitxml=junit/test-results.xml --cov mbed_targets --cov-report xml:coverage/coverage.xml --cov-report=html
```

Comment:
Welp, I did remove `html` generation. Guess that'll be useful for local development. 👍 

Why is `junitxml` necessary?

Reply:
It was the recommended command line for Python, thought it might help with the warnings and be generally useful.